core/src/main/java/com/linecorp/armeria/server/metric/PrometheusExpositionService.java

```diff
@@ -43,8 +44,6 @@
  */
```

Comment:
Could you update the Javadoc? 😉
https://github.com/line/armeria/blob/198c247f2e9092601e920f900412f3cd1c70d7e0/core/src/main/java/com/linecorp/armeria/server/metric/PrometheusExpositionService.java#L41-L42